src/ContinuityMassElemKernel.C

```diff
@@ -61,7 +61,7 @@ ContinuityMassElemKernel<AlgTraits>::ContinuityMassElemKernel(
   dataPreReqs.add_cvfem_volume_me(meSCV);
 
   // fields and data
-  dataPreReqs.add_gathered_nodal_field(*coordinates_, AlgTraits::nDim_);
+  dataPreReqs.add_coordinates_field(*coordinates_, AlgTraits::nDim_);
```

Comment:
By add_coordinates_field(), you mean set the coordinates to be used for standard grad_op and areavec calcs?

As long as we support the ability to have a grad_op (based on model) and grad_op (based on current) in addition to a coordinates field used in the kernel itself, i.e., coord(nodeL) - coord(nodeR), I am fine:)